services/src/main/java/org/exoplatform/news/NewsService.java

```diff
@@ -25,6 +25,30 @@
    */
   News getNewsById(String newsId, boolean editMode);
 
+  /**
+   * Get the news draft visibility
+   *
+   * @param newsId {@link News} identifier
+   * @return {@link String} The news draft visibility
+   */
+  String getNewsDraftVisibility(String newsId) throws Exception;
+
+  /**
+   * Is news author
+   *
+   * @param newsId {@link News} identifier
+   * @return {@link String} Is news author
+   */
+  boolean isNewsAuthor(String newsId, String currentUser) throws Exception;
+
+  /**
+   * Set the news draft visibility
+   *
+   * @param newsId {@link News} identifier
+   * @param draftVisibility {@link News} The draft visibility
+   */
+  void setNewsDraftVisibility(String newsId, String draftVisibility) throws Exception;
```

Comment:
not needed IMHO since you can set this information in news object by calling updateNews rest service